CMakeLists.txt

```diff
@@ -7,9 +7,15 @@ find_package(Swift CONFIG REQUIRED)
 find_package(Clang CONFIG REQUIRED)
 
 set(SwiftREPL_TESTS_DIR ${CMAKE_SOURCE_DIR}/tests)
+set(LIT ${LLVM_TOOLS_BINARY_DIR}/llvm-lit.py)
 
 configure_file(${SwiftREPL_TESTS_DIR}/lit.cfg.py.in ${SwiftREPL_TESTS_DIR}/lit.cfg.py)
-configure_file(test.py.in test.py)
+
+if(NOT DEFINED LIT_ARGS_DEFAULT)
+  set(LIT_ARGS_DEFAULT -v -vv)
```

Comment:
Why are these the default args btw?

Reply:
Personal preference - I like it it showing more information. Plus, if a statement fails, it'll immediately print it

Reply:
In general that slows down testing, so it is better to let it only print out the data if needed (which it does if the test fails).